source/js/foundation/pages/youtube-regrets/regrets-reporter/share-buttons.js

```diff
@@ -0,0 +1,113 @@
+import { gsap } from "gsap";
+import copyToClipboard from "../../../../copy-to-clipboard";
+let tl = gsap.timeline();
+
+class regretsReporterShareButtons {
+  constructor() {
+    this.shareButtonsActive = false;
+    this.bindEvents();
+  }
+
+  bindEvents() {
+    const shareDisplayButton = document.getElementById("share-display-button");
+    const shareFacebookButton = document.getElementById("share-fb-button");
+    const shareTwitterButton = document.getElementById("share-twitter-button");
+    const shareEmailButton = document.getElementById("share-email-button");
+    const shareLinkButton = document.getElementById("share-link-button");
+
+    shareDisplayButton.addEventListener("click", () => {
+      if (!tl.isActive()) {
+        this.updateShareButtons();
+      }
+    });
+
+    shareFacebookButton.addEventListener("click", (e) => {
+      this.shareButtonClicked(e, "share-progress-fb");
+    });
+
+    shareTwitterButton.addEventListener("click", (e) => {
+      this.shareButtonClicked(e, "share-progress-tw");
+    });
+
+    shareEmailButton.addEventListener("click", (e) => {
+      this.shareButtonClicked(e, "share-progress-em");
+    });
+
+    shareLinkButton.addEventListener("click", (e) => {
+      this.shareButtonClicked(e);
+    });
+  }
+
+  // Clicks the hidden Shareprogress buttons on extension_hero.html,
+  // or copies URL to clipboard.
+  shareButtonClicked(event, shareProgressButtonId) {
+    if (shareProgressButtonId) {
+      let shareProgressButton = document.querySelector(
+        `#${shareProgressButtonId} a`
+      );
+
+      if (shareProgressButton) {
+        shareProgressButton.click();
+      }
+    } else {
+      navigator.clipboard.writeText(window.location.href);
+      event.target.classList.add(`copied`);
+    }
+  }
+
+  updateShareButtons() {
+    if (this.shareButtonsActive) {
+      this.hideShareButtons();
+    } else {
+      this.displayShareButtons();
+    }
+    this.shareButtonsActive = !this.shareButtonsActive;
+  }
+
+  // Display share buttons and hide "Share This" text
+  displayShareButtons() {
+    tl.to("#share-display-text", {
+      display: "none",
```

Comment:
You can't 'animate' display. You want to animate the text and then add the the display none afterwards. Remove `display: "none",` from `tl.to` and then add a line afterwards using `tl.set` instead. If you do not want the element to be animated then just remove the other properties besides display none and use `tl.set` instead of `tl.to`

Reply:
Updated!